pfcpiface/bess.go

```diff
@@ -18,6 +20,9 @@ import (
 	"google.golang.org/protobuf/types/known/anypb"
 )
 
+// SockAddr : Unix Socket path to read bess notification from
+const SockAddr = "/tmp/notifycp"
```

Comment:
Is this configurable with this as the default?

Reply:
done